exp/etcdrestore/main.go

```diff
@@ -219,47 +218,47 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
 		os.Exit(1)
 	}
 
-	if feature.Gates.Enabled(feature.EtcdSnapshotRestore) {
```

Comment:
with this removed, are we allowing to run the etcdsnapshot controllers always by default?